silence-removal/vad.py

```diff
@@ -143,28 +134,72 @@ def get_audio_channels(self, data):
         else:
             raise Exception("Error, data is neither mono or stereo!")
 
-# Usage:
-# Convert the `.aac` audio file into `.wav`
-input_filename = sys.argv[1]
-audio_file_extension = Path(input_filename).suffix
-logging.info(f"Audio file extension: {audio_file_extension}")
-if audio_file_extension != ".wav":
-    convert_to_wav(input_filename)
-    # replace extension with .wav
-    newFilename = re.sub(pattern = "\.mp3", repl = ".wav",
-                         string = input_filename)
-else:
-    newFilename = input_filename
-wav = wf.read(newFilename)
-sample_rate = wav[0]
-audio_data = wav[1]
-logging.info(f"Audio data: {audio_data}")
-
-vad = VoiceActivityDetection()
-# Remove the silences
-vad.process(audio_data)
-# Get the processed audio
-voice_samples = vad.get_voice_samples()
-output_name = sys.argv[2]
-# Write the processed audio into a new `.wav` file
-wf.write(f"{output_name}.wav", sample_rate, voice_samples)
+
+def remove_silences(date_folder):
+    """
+    Function that removes silences from all audio files in a folder.
+    File structure is
+    ```
+    | data
+    | |-- reddit
+    |     |-- date_folder
+    |         |-- filename_folder
+    |             |-- audio_file
```

Comment:
probably make this plural - audio_fileS

Reply:
I've [updated the function documentation](https://github.com/myHaiven/data-collection/pull/4/commits/c3f23c5b130c25125ffad2db45f7ffa662bff7ad) so that audio files is now plural.